network/src/commonMain/kotlin/com/wire/kalium/network/api/teams/TeamsApi.kt

```diff
@@ -13,26 +13,15 @@ interface TeamsApi {
 
     @Serializable
     data class TeamsResponse(
-        @SerialName("has_more") val hasMore: Boolean,
-        val teams: List<Team>
+        @SerialName("has_more") val hasMore: Boolean, val convTeamInfos: List<TeamDTO>
     )
 
-    @Serializable
-    data class Team(
-        val creator: String,
-        val icon: AssetId,
-        val name: String,
-        val id: TeamId,
-        @SerialName("icon_key") val iconKey: AssetId?,
-        val binding: Boolean
-    )
 
     @Serializable
     data class TeamMemberList(
         // Please note that this is intentionally cased differently form the has_more in TeamsResponse
         // because the backend response contains a different casing
-        @SerialName("hasMore") val hasMore: Boolean,
-        val members: List<TeamMember>
+        @SerialName("hasMore") val hasMore: Boolean, val members: List<TeamMember>
```

Comment:
Small suggestion on code format:
```suggestion
        @SerialName("hasMore") val hasMore: Boolean,
        val members: List<TeamMember>
```